docs/TSG.md

```diff
@@ -224,6 +225,44 @@ Chromium-based browsers requires the server certificate to be trusted by a defau
 
 See [Chromium network switches](https://source.chromium.org/chromium/chromium/src/+/main:services/network/public/cpp/network_switches.cc;l=36;drc=f8c933c2bd17344ce7ac61be2ac7725ed840b19f)
 
+## Collecting a Packet Capture
+
+### Linux Packet Capture
+
+> TODO
+
+### Window Packet Capture
+
+On Windows, [Packet Monitor](https://docs.microsoft.com/en-us/windows-server/networking/technologies/pktmon/pktmon?msclkid=79c406dcab7711ec976873fd5c4a48bf) (`pktmon`) is the best way to collect a packet capture.
+
+The first step is usually to find the interface you want to collect the capture on. Do this by first running `pktmon list`:
+
+```
+> pktmon list
+
+Network Adapters:
+   Id MAC Address       Name
+   -- -----------       ----
+    9 40-8D-5C-B5-46-51 Intel(R) Ethernet Connection (2) I219-V #2
+  104 00-15-5D-D1-5A-30 433db3ea-0acd-457a-9c86-55bb7fa27391
+   80 00-15-5D-AD-8B-40 433db3ea-0acd-457a-9c86-55bb7fa27391
+```
+
+Once you find the interface you want, take note of the `Id`. For instance, in the example above, I want to use the Ethernet adapter, so I need `9`.
+
+Then, to collect the capture for your scenario (example below uses port `443`), run the following:
+
+```
```

Comment:
Another way to do this is to perform the packet capture for all NIC components, and then filter by NIC at the etl2pcap step.  This allows the user to do 1 packet capture step for the whole system.

```
pktmon filter remove
pktmon filter add -t UDP -p 443
pktmon start --capture --comp nics --pkt-size 0 
<run scenario>
pktmon stop
pktmon etl2pcap pktmon.etl --component-id 9 --out pktcap_if9.pcapng
```